src/test/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/templates/AppealSubmissionTemplateTest.java

```diff
@@ -249,6 +251,7 @@ void dataSetUp() {
         when(asylumCase.read(LEGAL_REP_REFERENCE_NUMBER, String.class)).thenReturn(Optional.of(legalRepReferenceNumber));
         when(asylumCase.read(HOME_OFFICE_REFERENCE_NUMBER, String.class)).thenReturn(Optional.of(homeOfficeReferenceNumber));
         when(asylumCase.read(HOME_OFFICE_DECISION_DATE, String.class)).thenReturn(Optional.of(homeOfficeDecisionDate));
+        when(asylumCase.read(DECISION_LETTER_RECEIVED_DATE, String.class)).thenReturn(Optional.of(decisionLetterReceivedDate));
```

Comment:
@JamieHMCTS , is there a test that when reading DECISION_LETTER_RECEIVED_DATE or HOME_OFFICE_DECISION_DATE, they returns null or "" ? So that it does not go into formatDateForRendering